src/QsCompiler/Core/Utils.fs

```diff
@@ -1,7 +1,8 @@
-﻿module internal Microsoft.Quantum.QsCompiler.Utils
+﻿module Microsoft.Quantum.QsCompiler.Utils
 
 /// Converts a C# Try-style return value into an F# option.
-let tryToOption =
+[<CompiledName "TryOption">]
```

Comment:
Since this is now public, should the comment above be enclosed in `<summary></summary>` tags?

Reply:
No, `<summary>` is implicit in F#. It's only needed when other documentation tags are used.

Reply:
Thanks for the explanation!